### ISSUE TITLE
Typo in Ring of Wealth

### ISSUE THREAD

**Issue by Jordy3D** [mythridium/item-descriptions-expanded#2]
I believe 
![image](https://github.com/mythridium/item-descriptions-expanded/assets/19144524/7c6c2736-5dcd-405d-b415-dbe3d9435957)
should read something like
```
Rumors of this ring's ability to attract wealth and create gold has spawned countless knockoffs. Some successful, others not so much.
```

I checked GPT Japanese and GPT German, and both of them seemed to have been correctly corrected, so this likely just affects English.

**Comment by mythridium** [comment]
Thanks for the report, I actually noticed the same and have fixed this locally, I was waiting for the next set of descriptions so I can include it with a larger change.

GPT seems to be fairly good at picking up these mistakes when doing the translations.

If you spot anymore, please let me know and I'll get them fixed.

I've currently got the following fixed locally:
- Ancient Ring of Mastery
- Wasteful Ring
- Ring of Wealth

**Comment by Jordy3D** [comment]
I spat out every entry into a JSON file for me to look through, and found a few to consider:

| Item | Current | Fix | Note |
|-|-|-|-|
| Ice_Arrows | An Arrow embed with the magic of the Deep South. Careful how you handle them. | An **arrow embedded** with the magic of the Deep South. Careful how you handle them. |
| Jadestone_Bolts | Iron crossbow bolts tipped with splintered Jadestone. Legend says they were designed by a Princess to kill her Dragon captor. | Iron crossbow bolts tipped with splintered Jadestone. Legend says they were designed by a Princess to kill her **dragon** captor. | 
| Zephyte_Bolts | Iron crossbow bolts tipped with a splintered gem of Divine origin. Destructive as it is beautiful. | Iron crossbow bolts tipped with a splintered gem of **divine** origin. **As destructive** as it is beautiful. |
| Amulet_of_Defence | Amulet enchanted to harden your skin and protect the wearer from damage. Need some lotion? | **An amulet** enchanted to harden your skin and protect the wearer from damage. Need some lotion? | The other Amulets from this group start with "An" or "A", so it's for consistency.
| Amulet_of_Looting | Wearing this marks you as the Chief of the Pygmy spiders. Force them to bank your stuff. | Wearing this marks you as the Chief of the Pygmy **Spiders**. Force them to bank your stuff. |
| Silver_Emerald_Necklace | Silver and emeralds are said to bring good luck. Both in finding lost arrows... and ensuring they find your enemy. | Silver and emeralds are said to bring good **luck, both** in finding lost arrows... and ensuring they find your enemy. |
| Silver_Diamond_Necklace | Why do silver and diamonds make you hit things harder with bit sticks? No one knows. It is one of the great mysteries of Melvor. | Why do silver and diamonds make you hit things harder with **big** sticks? No one knows. It is one of the great mysteries of Melvor. |
| Warlock_Amulet | The warlocks of Glacia city were said to drain the life out of their foes. All who wore this amulet eventually succumb to the Burning Madness. | The warlocks of Glacia city were said to drain the life out of their foes. All who wore this amulet eventually **succumbed** to the Burning Madness. | This could also be "All who wear this ... eventually succumb". I just tried matching tense. |
| Amulet_Of_Incantation | Though fragile, thus unsuited for battle, this enchanted Amulet is popular among spell casters for its ability to increase the caster's speech. | Though fragile, thus unsuited for battle, this enchanted **amulet** is popular among spellcasters for its ability to increase the caster's speech. |
| Palladium_Onyx_Necklace | A favorite of the Hunters the world around. This amulet increases your hand-eye coordination, making it easier to harvest valuable materials from slain foes. | A **favorite of hunters** the world over. This amulet increases your hand-eye coordination, making it easier to harvest valuable materials from slain foes. |
| Palladium_Cerulean_Necklace | The powerful enchantment life enchantment on this stone has been tweeted to drastically increase your healing speed, but at the cost of more bodily resources. | The **powerful life enchantment** on this stone has been **tweaked** to drastically increase your healing speed, but at the cost of more bodily resources. |
| Poison_Lifesteal_Amulet | Anything can be a medicine in the right dose. Even poison. | Anything can be a medicine **with** the right dose. Even poison. | Alternatively, "in the right amount" |
| Frostshield_Amulet | An Amulet carved from the cold heart of a Ice Hydra. Its magic grants you some of the creature's power when surrounded by ice. | An amulet carved from the cold heart of **an** Ice Hydra. Its magic grants you some of the creature's power when surrounded by ice. |
| FrostSpark_Amulet | By sating the Lord of the Sky's grudge with the chilling blood of the Ice Hydra, you have unlocked true power of this amulet. | By sating the Lord of the Sky's grudge with the chilling blood of the Ice Hydra, you have unlocked **the** true power of this amulet. |
| Amulet_of_Curse_Totem_Summon | A Great Spirit was once split into three parts by an evil Necromancer. This Amulet was crafted from its body. Anyone who meets its gaze will be cursed for eternity. | A Great Spirit was once split into three parts by an evil Necromancer. This **amulet** was crafted from its body. Anyone who meets its gaze will be cursed for eternity. |
| Dragon_Arrowtips | For every great foe slain, an Arid Plains Chieftain will add a Dragonite Arrowtip to their headpiece. Both as a sign of honor... and a warning. | For every great foe slain, an Arid Plains Chieftain will add a **dragonite** arrowtip to their headpiece, both as a sign of honor... and a warning. | Brings the Dragon Arrowtips into capitalised sync with the others. See the section beneath the table for that discussion too. I'd actually vote for them all to be capitalised, but for now... consistency among Arrowtips. |
| Corundum_Arrowtips | Arrowtips forged from Corundum metal are so sharp that they can leave debilitating, lingering wounds. | Arrowtips forged from **corundum** metal are so sharp that they can leave debilitating, lingering wounds. | See above. |
| Thiefs_Moneysack_II | This enchanted satchel attracts gold like a magnet! Regardless if said gold belongs to you or not. | This enchanted satchel attracts gold like a **magnet, whether** said gold belongs to you or not! |
| Seed_Pouch_II | Vr. 2.0 from your friendly neighbors at Seed Co.! Now less* likely than ever to attract ravenous hordes of monstrous insects! | **Ver**. 2.0 from your friendly neighbors at Seed Co.! Now less* likely than ever to attract ravenous hordes of monstrous insects! |
| Stamina_Pouch | A small pouch of white pills said to boost Stamina and increase agility. Warning, if you find yourself on the obstacle course for more than 4 hours, consult a doctor. | A small pouch of white pills said to boost Stamina and increase agility. **Warning:** if you find yourself on the obstacle course for more than 4 hours, consult a doctor. |
| Meteorite_Marksman_Ring | An Archer once dreamed of shooting the very stars from the sky. When those stars fell, his enemies have no place to hide. | An Archer once dreamed of shooting the very stars from the sky. When those stars fell, his enemies **had** no place to hide. |
| Deadeye_Ring | A signet ring marking Aeris' Generals in a long ago war. The gem at its center creates a channel of air that increases an arrows speed and power. | A signet ring marking Aeris' Generals in a long ago war. The gem at its center creates a channel of air that increases an **arrow's** speed and power. |
| Guardian_Ring | The Sentinels of Terran wore these rings during their eternal vigilance. Its magic encases the wearer in a thick later of hard stone that can block any blade. | The Sentinels of Terran wore these rings during their eternal vigilance. Its magic encases the wearer in a thick **layer** of hard stone that can block any blade. |
| Fighter_Ring | Gifted to the Champions of Ragnar's Pits, gladiators who wore these rings were rumored to be more beast than man, and revealed in the combat. | Gifted to the Champions of Ragnar's Pits, gladiators who wore these rings were rumored to be more beast than man, and **revelled** in the combat. |
| Silver_Topaz_Ring | While not as potent as a larger amulet, this ring is said to help guide true the trust of an errant blade. | While not as potent as a larger amulet, this ring is said to help guide true the **thrust** of an errant blade. |
| Silver_Emerald_Ring | A bow drawn or object thrown with hand wearing this ring will fly farther and strike harder than normal. | A bow drawn or object **thrown by a hand** wearing this ring will fly farther and strike harder than normal. |
| Gold_Emerald_Ring | Used by would-be-knights to train their skills, a weapon swung with a hand wearing this ring will feel far heavier. | Used by **would-be** knights to train their skills, a weapon swung with a hand wearing this ring will feel far heavier. |
| Gold_Diamond_Ring | Those who wear this ring need never fear misplacing their things, and as such it has been celebrated as good luck charm since time immemorial. | Those who wear this ring need never fear misplacing their things, and as such it has been celebrated as **a** good luck charm since time immemorial. |
| Ancient_Ring_Of_Skills | This enchanted ring can manifest a variety of tools that would make any craftsman salivate. They are exceedingly rare, so only a few have ever even see one. | This enchanted ring can manifest a variety of tools that would make any craftsman salivate. They are exceedingly rare, so only a few have ever even **seen** one. |
| Sandstorm_Ring | The slaying of a Sand Beast and fashioning of its heart into this ring is one of the most dangerous rights of passage for any young Arid Plains Tribesman... and the most respected. | The slaying of a Sand Beast and fashioning of its heart into this ring is one of the most dangerous **rites** of passage for any young Arid Plains Tribesman... and the most respected. |
| Natures_Blessing_Ring | A Shaman's ring blessed by their Drudic allies as a sign of their pact. Brings one closer to nature and improves their ability to heal. Both welcomed in the harsh Badlands. | A Shaman's ring blessed by their **Druidic** allies as a sign of their pact. Brings one closer to nature and improves their ability to heal. Both welcomed in the harsh Badlands. |
| Ring_Of_Barrage | The vines growing around this ring secret a potent stimulant that can sharpen a wear's reflexes, but makes them more prone to mistakes. | The vines growing around this ring **secrete** a potent stimulant that can sharpen a wear's reflexes, but makes them more prone to mistakes. |
| Palladium_Onyx_Ring | This ring produces powerful magical shield designed to deflect fast moving projectiles from a bow or crossbow. | This ring produces **a** powerful magical shield designed to deflect fast moving projectiles from a bow or crossbow. |
| Iridium_Cerulean_Ring | Originally created with the idea of erasing hunger by "feeding" on magical energy, the experiment was deemed a failure. Regardless its effects are still potent. | Originally created with the idea of erasing hunger by "feeding" on magical energy, the experiment was deemed a failure. **Regardless,** its effects are still potent. |
| Iridium_Runestone_Ring | When swung, a weapon by a hand wearing this ring produces a magical blade that drastically increases its power. | **A weapon, when swung by a hand wearing this ring**, produces a magical blade that drastically increases its power. |
| Palladium_Runestone_Ring | Building on the Silver Sapphire ring, When powered, this ring produces a small shield of energy around the wear's forearm. This shield is far sturdier and more reliable. | Building on the Silver Sapphire ring, **this ring produces a small shield of energy around the wear's forearm when powered**. This shield is far sturdier and more reliable. |
| Ring_of_Curse_Protection | A ring granted to servants of Lady Darkheart. Mostly to protect them from their own power. | A ring granted to servants of Lady **Darkheart, mostly** to protect them from their own power. | Alternatively, "...Darkheart. Mostly protects them from..." |
| Blood_Ring | A ring used by Vampires to gather the blood of their victims. Such greedy creatures can't bear to waste even a single drop. | A ring used by **vampires** to gather the blood of their victims. Such greedy creatures can't **bare** to waste even a single drop. |
| Book_of_the_Ancients | The complete works of an Ancient, magical civilization lost to time. Once broken apart for fear of what this knowledge might unleash, it is once more whole. | The complete works of an Ancient, magical civilization lost to time. Once broken apart for fear of what this knowledge might unleash, it is **once again whole**. |
| Golden_Star_II | Alchemically treated, grinding these gems can produces far more celestial power than normal. | Alchemically treated, grinding these gems can **produce** far more celestial power than normal. | I'm not sure if "treated" should be "created", either. |
| Fishing_Scroll_Of_XP | An old sailor sold you this scroll. Inside are descriptions of every fish he'd every caught, from the mist shrouded north to the frozen south. | An old sailor sold you this scroll. Inside are descriptions of every fish he'd every caught, from the **mist-shrouded** **North** to the frozen South. |
| Crafting_Scroll_Of_XP | This scroll contains various blueprints for a wide range of common project to help any artisan polish their skills. | This scroll contains various blueprints for a wide range of common **projects** to help any artisan polish their skills. |
| Monster_Hunter_Scroll | Half the challenge in monster hunting, is finding them. This scroll lists the habitats and wandering range of various creatures. | Half the challenge in monster **hunting... is finding them**. This scroll lists the habitats and wandering range of various creatures. | swapped the errant comma for an ellipses to keep the (I assume) intended pause. If that's not intended, just drop the comma. |
| Mastery_Magnet | A mysterious magnet said to attract Knowledge. Its unknown if it actually works, but many Artisans swear by them. | A mysterious magnet said to attract Knowledge. **It's** unknown if it actually works, but many Artisans swear by them. |
| Bad_Cooker_Scroll | Maybe this new Assistant Chef you hired isn't quite as efficient as you were lead to believe... | Maybe this new Assistant Chef you hired isn't quite as efficient as you were **led** to believe... |
| Gem_Finder_Scroll | This magical paper detects any hidden precious stones of a specific size and quality, when a piece of ore is placed on top. | This magical paper detects any hidden precious stones of a specific size and **quality when** a piece of ore is placed on top. |
| Slayer_Harvest_Scroll | Bloodstones produced by this scroll are useless to most, but fetch a decent price in the slayer lodge. I wound who's buying them? | Bloodstones produced by this scroll are useless to most, but fetch a decent price in the **Slayer Lodge**. I wonder **who's** buying them? | Also matches the capitalised Slayer Lodge of some of the other Scrolls |
| Magic_Hinder_Scroll | A Trap scroll favored by hunters of the Slayer Lodge. When placed on a creature's hide, these sticky scrolls attracts magical blasts to their location. | A Trap scroll favored by hunters of the Slayer Lodge. When placed on a creature's hide, these sticky scrolls **attract** magical blasts to their location. |
| Melee_Hinder_Scroll | A Trap scroll favored by hunters of the Slayer Lodge. When trigger, this scroll sprays the target with a mild acid, weakening their hide or armor. | A Trap scroll favored by hunters of the Slayer Lodge. When **triggered**, this scroll sprays the target with a mild acid, weakening their hide or armor. |
| Fishing_Hook | H-have you just been fishing with an unhooked like this whole time?! No wonder this is taking so long... | H-have you just been fishing with an **unhooked line** like this whole time?! No wonder this is taking so long... |
| Wizards_Scroll | Some wizards of the Academia Arcanum have spend their entire lives optimizing their spells. This is the crystallization of all their effort. | Some wizards of the Academia Arcanum have **spent** their entire lives optimizing their spells. This is the crystallization of all their **efforts**. |
| Explorers_Map_II | A map commissioned from a Veteran Explorer. The level of detail and information contained here boggled your mind. | A map commissioned from a Veteran Explorer. The level of detail and information contained here **boggles** your mind. |
| Portable_Rations | Specially designed for those traveling long distances, each pack is its own self contained meal. | Specially designed for those traveling long distances, each pack is its own **self-contained** meal. |
| Blood_Vial | Jokingly called a "Health Potion" in the vampire community, this thick red solution isn't truly blood, but a blood substitute used to curve the Hunger. | Jokingly called a "Health Potion" in the vampire community, this thick red solution isn't truly blood, but a blood substitute used to **curb** the Hunger. |
| Adamantite_Ore | Small grains of Adamantite can be shifted from the sands of the Arid Plains and its Desolate north. Rumor speaks of a grand mine somewhere under the sands. | Small grains of Adamantite can be **sifted** from the sands of the Arid Plains and its Desolate north. Rumor speaks of a grand mine somewhere under the sands. | Unsure. Shifted works, but I have a feeling that sifted was intended. |
| Corundumite_Ore | This ore was once thought useless, until a innovative technique by a young blacksmith unlocked its true potential. The gold-rush that followed made many men, and broke many more. | This ore was once thought useless, until **an** innovative technique by a young blacksmith unlocked its true potential. The gold-rush that followed made many men, and broke many more. |
| Iridium_Ore | Wars have been fought over small deposits of this mineral. In modern times, deposits, both known and unknown are considered the exclusive property of the Crown. | Wars have been fought over small deposits of this mineral. In modern times, deposits, both known and **unknown, are** considered the exclusive property of the Crown. |
| Palladium_Ore | The few location where this rare, magically conductive metal can be can be found are jealously guarded. More so is the method to properly work it. | The few **locations** where this rare, magically conductive metal can be can be found are jealously guarded. More so is the method to properly work it. |
| Meteorite_Ore | What happens with a star dies? This ore is filled with vast amounts of Celestial power, but only those who regularly study the heavens have any hope of ever finding it. | What happens **when** a star dies? This ore is filled with vast amounts of Celestial power, but only those who regularly study the heavens have any hope of ever finding it. |
| Divinite_Ore | No one quite knows where this ore comes from. It appears seemingly at random in abandon mines otherwise long empty. Over time, people just attributed it to the whim of the gods. | No one quite knows where this ore comes from. It appears seemingly at random in **abandoned mines that are otherwise long empty**. Over time, people just attributed it to the whim of the gods. |
| Bronze_Bar | Legend says the Golbin were the first people learn how to work bronze. While that is up for debate, none can deny that this metal's history is as long as the world itself. | Legend says the Golbin were the first **people to learn** how to work bronze. While that is up for debate, none can deny that this metal's history is as **old** as the world itself. |
| Iron_Bar | Versatile, durable and easy to work, Iron was and is the foundation on which society in Melvor has developed. | Versatile, **durable, and easy to work**, Iron was and is the foundation on which society in Melvor has developed. |
| Mithril_Bar | Highly sough after for its ability to conduct holy power, the Holy Isles guard its supply of Mithril with a fervor that has repelled countless assaults. | Highly **sought** after for its ability to conduct holy power, the Holy Isles guard its supply of Mithril with a fervor that has repelled countless assaults. |
| Adamantite_Bar | Adamantite plays a huge role in the culture and lifestyle of the Arid Plains people. When no warbands are called, many tribesmen many a living shifting Adamantite flakes from the sands. | Adamantite plays a huge role in the culture and lifestyle of the Arid Plains people. When no warbands are called, many tribesmen **make** a living **sifting** Adamantite flakes from the sands. |
| Dragonite_Bar | Eastern merchants brought Dragonite ore to the west as a curiosity. Until it was found alloying the ore with western Runite vastly increased both metal's potential. | Eastern merchants brought **dragonite ore** to the West. **It was initially considered just a curiosity, until** it was found **that** alloying the ore with western Runite vastly increased both metal's potential. | Alternatively, **Dragonite Ore**
| Palladium_Bar | Palladium is prized for its ability to conduct magical energy and is theorized to the be the key in many lost technologies. Scholars are only now relearning its secrets. | Palladium is prized for its ability to conduct magical energy and is theorized to be the key **to** many lost technologies. Scholars are only now relearning its secrets. |
| Topaz | When charged with magical energy, Topaz can increase a person's reflexes or luck, depending if its paired with silver or gold. Some question if these are not simply the same thing. | When charged with magical energy, Topaz can increase a person's reflexes or luck, **depending on whether** it's paired with silver or gold. Some question if these are not simply the same thing. |
| Diamond | Mostly thought of as pretty rock, recent magical advancements have show these gemstones have the peculiar ability to affect probability. What are the chances? | Mostly thought of as pretty rock, recent magical advancements have **shown** these gemstones have the peculiar ability to affect probability. What are the chances? |
| Necromancer_Boots | The Ancient Necromancer who founded the school was said to be one of the most powerful ever born... or since. | The Ancient Necromancer who founded the school was said to be one of the most **powerful of their day... or since**. | "ever born" and "since" is kinda the same thing.
| Summoning_Shard_Red | The bloody shard of a wayward soul. Red shards resonate strongest with bloodshed and violence. Be it protection from, or the perpetuation of it. | The bloody shard of a wayward soul. Red shards resonate strongest with bloodshed and **violence, be it** protection from, or the perpetuation of it. |
| Summoning_Shard_Green | A vibrant shard of a wayward soul. Green shards resonate best with the Spirits of natural creatures. Both animals and elemental Spirits. | A vibrant shard of a wayward soul. Green shards resonate best with the Spirits of natural **creatures; both** animals and elemental Spirits. |
| Summoning_Shard_Silver | This shard glistens in the light. Silver shards are said to attract Spirits of wisdom and fortune, thus are popular among craftsmen and scholars of all types. | This shard glistens in the light. Silver shards are said to attract Spirits of wisdom and fortune, **and thus** are popular among craftsmen and scholars of all types. |
| Summoning_Familiar_Ent | These tree inhabiting spirits regularly shed dead branches and improve the heath of their host tree. Wood shed by an Ent is highly sought after. | These **tree-inhabiting** spirits regularly shed dead branches and improve the **health** of their host tree. Wood shed by an Ent is highly sought after. |
| Summoning_Familiar_Mole | These earth spirits, while appearing as a common mole, have the canny ability to sniff out mineral deposits. Though feeding them may cost more than new summoners can afford. | These earth spirits, while appearing as a common mole, have the canny ability to sniff out mineral deposits. **However, feeding** them may cost more than new summoners can afford. |
| Summoning_Familiar_Occultist | These blind hags have only one desire in life. Twisting the mind of the weak-willed. Point them at your foe and they will gladly work their magics. | These blind hags have only one desire in **life: twisting** the mind of the weak-willed. Point them at your foe and they will gladly work their magics. |
| Summoning_Familiar_Pig | A fable tells the story of a master chef who refused to share his food with anyone else. In punishment, the gods transformed him into a pig. undeterred, the chef continued his work. | A fable tells the story of a master chef who refused to share his food with anyone else. In punishment, the gods transformed him into a pig. **Undeterred**, the chef continued his work. |
| Summoning_Familiar_Crow | Crows have long been favored familiars of wizards, for their natural gifts in channeling magical energies. Whither by instinct or innate intelligence, the crows aren't saying. | Crows have long been favored familiars of wizards, for their natural gifts in **channelling** magical energies. **Whether** by instinct or innate intelligence, the crows aren't saying. |
| Summoning_Familiar_Centaur | Fierce competitors to the Minotaurs, the bloody wars between these two ground has shifted into more one of competition other the years. As Mercenaries, they now compete with gold instead. | Fierce competitors to the Minotaurs, the bloody wars between these two **groups** has shifted more into one of competition other the years. As Mercenaries, they now compete with gold instead. | Honestly not sure, but "ground" ain't right. |
| Summoning_Familiar_Witch | Its said that a witch's soul can be trapped if the proper steps are taken. Whether or not this is advisable, Is up for debate, but some are beyond such petty concerns. | It's said that a witch's soul can be trapped if the proper steps are taken. Whether or not this is **advisable is up for debate**, but some are beyond such petty concerns. |
| Summoning_Familiar_Cyclops | Cyclops culture revolves around the hunt, and as such | Cyclops culture revolves around the hunt, and as such **they are happy to be summoned to fight once more.** | This one just... ended out of nowhere. |
| Summoning_Familiar_Salamander | These Spirits of Fire love to nap in well tended furnaces. Smiths try to attract them, as they cleanse the strengthen the flames within, resulting in higher quality metals. | These Spirits of Fire love to nap in well-tended furnaces. Smiths try to attract them, as they **purify and strengthen** the flames within, resulting in higher quality metals. |

Additionally, all the Bolts should probably either have their gem (Ruby, Topaz, etc) capitalised, or al lthe Arrows should have their metal (Steel, Mithril, etc) decapitalised, for consistency.
This would also apply to things like the Silver Topaz Necklace and Ring (and the rest of them), which also lack capitalised materials the way the arrows do. Interestingly, the Gold Sapphire necklace follows the arrows.
Probably goes for Ores and stuff too.

Should I change the title of this Issue to "Typos" and keep it open for future posts?


EDIT:
Bolded my changes/additions/etc.

**Comment by mythridium** [comment]
Thanks for this list! This is awesome.

I think you can leave the title as is, I'll close this issue once I've got these changes in, it's easier to track if you open another issue for any further ones.

**Comment by mythridium** [comment]
For future, could I request that in the fix column you bold or cross out the additions/deletions, that would make it easier to track down the difference. :) If it isn't too much trouble.

**Comment by Jordy3D** [comment]
Haha, good idea. I'll go through and edit the above table now as well

**Comment by Jordy3D** [comment]
Bolded my changes. They're a tad hard to see on GitHub though, and underline doesn't work, so I hope that's at least useful enough

**Comment by mythridium** [comment]
Thanks so much, I've just released these edits in v0.4.2

I only fixed English for the moment since the files are generated from a script I wrote.

The translations will have to wait for another time as I'm about to head to bed and it takes some time to push these through chat-gpt since I'm not paying for the api, I have to use the chat and feed it through manually.

I'll close this issue for now, but feel free to open any further issues as needed.

I'm thinking I might just open new issues as TheRealOsamaru releases them for you to review before I add them. It would be easier to translate them after being reviewed instead of doing the translations twice. https://www.reddit.com/r/MelvorIdle/comments/16nccxv/im_writing_a_description_for_every_item_in_the/

Thanks so much for your help, I've added you as a contributor on the mod page and the readme!

Fixed in ccdb1723ee02a8dd0525fcab892a40fadf54b78a
